src/components/chips/demoValidation/index.html

```diff
@@ -0,0 +1,27 @@
+<div ng-controller="ChipsValidationCtrl as ctrl" layout="column" ng-cloak>
+  <md-content class="md-padding" layout="column">
+    <h2 class="md-title">Required</h2>
+    <form name="fruitForm">
+      <md-chips name="fruit"
+                ng-model="ctrl.selectedFruit"
+                ng-required="true"
+                placeholder="Add a fruit">
+      </md-chips>
+      <div ng-messages="fruitForm.fruit.$error">
+        <div ng-message="required">At least one fruit is required</div>
+      </div>
```

Comment:
These don't show up red because the `chips-theme.scss` is lacking styles for `md-invalid`. Can you please review the `input-theme.scss` and implement something similar for chips as part of this PR?

Reply:
`input.scss` also uses a `md-input-message-animation` class to animate the message in and to set a smaller line height and font size (among other things). Can we please add those styles to `chips.scss` so that they will be similar when used in a form together?

Reply:
The error message is showing up even when the input is pristine and not dirty. For input, this isn't the case. Can you please look into this?

Reply:
Ok, I found out what md-input-container is doing: It adds a special animation-class `md-input-messages-animation` (https://github.com/angular/material/blob/v1.1.7/src/components/input/input.js#L840) and does some stuff with it. I see two ways here. A: make the demos with md-chips inside md-input-container and assure that this have the same look&feel. B: Add a chips-special-class and add styles there. 
But B is not really following the ways it is made atm. ng-messages is just styled inside certain containers. So just A is left, but are chips belonging to md-input-container? - Is there any rule which forbids it?

Reply:
Yeah, I can see how we wouldn't want to style `ng-messages` that aren't contained in an AngularJS Material component. I've asked @DevVersion for some input and created a [CodePen](https://codepen.io/Splaktar/pen/QQmwop) to demonstrate how `ng-messages` are styled with chips today (with `md-max-chips`) for reference since there is no example in the demos.